app/views/lab_spaces/show.html.erb

```diff
@@ -362,7 +361,7 @@
           // Add the new equipment card
           const clone = document.importNode($equipoCardTpl[0].content, true);
           const $clone = $(clone);
-          $clone.find('.EquipoCard').data('id', newid).data('lsid', _lsid).data('lid', _lid);
+          $clone.find('.EquipoCard').attr('data-id', newid).attr('data-lsid', _lsid).attr('data-lid', _lid);
```

Comment:
Esto es lo mismo en efecto, any reason why you changed this?

Reply:
no estaba jalando, no se estaba cambiando y creí que esto podría ser también la razón por la que no jalaba el link cuando recién se creaba un nuevo card, pero como quiera tampoco jaló, la verdad no sé qué está fallando ahí

Reply:
Okok deja lo arreglo